R/MIRA.R

```diff
@@ -16,20 +16,21 @@
 #' @importFrom ggplot2 ggplot aes facet_wrap geom_boxplot geom_jitter geom_line
 #' @import BiocGenerics S4Vectors IRanges
 #' @importFrom data.table ":=" setDT data.table setkey fread setnames 
-#'             setcolorder melt setkeyv rbindlist setattr
+#'             setcolorder melt setkeyv rbindlist setattr setorder
 #' @importFrom Biobase sampleNames
 #' @importFrom bsseq getBSseq hasBeenSmoothed
 NULL
 
 
-# Because of some issues with CRAN submission,
+# Because of some issues with CRAN submission, 
```

Comment:
what exactly are you referring to? The section on `if(getRversion() >= "2.15.1")`? Unfortunately this is required to pass CMD check due to data.table quirks. See [this stackoverflow question](http://stackoverflow.com/questions/9439256/) for explanation.